readme.md

```diff
@@ -85,12 +85,12 @@ const fpPromise = FingerprintJS.load()
 
 📕 [Full documentation](docs/api.md)
 
-## Switch to [Pro version](https://fingerprintjs.com/github/) for free to get 99.5% identification accuracy
+## Upgrade to [Pro version](https://fingerprintjs.com/github/) to get 99.5% identification accuracy
 
 FingerprintJS Pro is a professional visitor identification service that processes all information server-side and transmits it securely to your servers using server-to-server APIs.
 Pro combines browser fingerprinting with vast amounts of auxiliary data (IP addresses, time of visit patterns, URL changes and more) to be able to reliably deduplicate different users that have identical devices, resulting in 99.5% identification accuracy.
 
-**You can use Pro FREE for up to 1,000 unique monthly visitors - no credit card required.**
+**You can try Pro without usage limits for 30 days- no credit card required.**
```

Comment:
Probably, a space is missing after the dash (see the before line)

Reply:
you are right, thanks @Finesse